### PR TITLE
Code review fixes and readme file

### PR DESCRIPTION
**Pull request by KamranIE**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,115 @@
...
+	1. Name e.g. 'LocalDev'
+	2. Publish Method: File System
+	3. Target location: <Umbraco web site e.g. C:\Projects\Acme\UmbracoCms.8.1.1>
+	4. Configuration: Debug
+### Acme.Umbraco.Models Project
+* It contains all the files from Umbraco models folder (e.g. \Acme\UmbracoCms.8.1.1\App_Data\Models in our case). 
+* __Important:__ The files from models folder are not automatically copied and included to the project. You will have to include any new changes to the project manually.
+ 
+---
+## Git setup
```
Comment (ettiennegous) [inline README.md]: I would remove this, its self documenting in the git file that is also checked in

File: src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    {
+        private List<AthleteViewModel> _athletes;
+
+        private PhysioAthletesViewModel(IPublishedContent containerItem, Physiotherapist physio, int maxAthletesToDisplay)
+        {
+            _athletes = new List<AthleteViewModel>();
+            MaxAthletesToDisplay = maxAthletesToDisplay;
+            if (containerItem != null)
+            {
+                SetUpBaseUrl(containerItem.Url);
```
Comment (ettiennegous) [inline src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs]: return and set

File: src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            SetupPhysioDataData(physio);
```
Comment (ettiennegous) [inline src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs]: return and set

File: src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        }
+
+        private void SetUpBaseUrl(string url)
+        {
+            if (url != null)
```
Comment (ettiennegous) [inline src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs]: Can you create a string extension method called .HasValue() 
Apply a couple of checks in it including null and empty values

File: src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+            get {
+                return _athletes;
+            }        
+        }
+
+        public IReadOnlyList<AthleteViewModel> DisplayableAthletes
+        {
+            get
+            {
+                return MaxAthletesToDisplay > 0 ? _athletes.Take(MaxAthletesToDisplay).ToList() : _athletes;
```
Comment (ettiennegous) [inline src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs]: What if the take is bigger than the list size

File: src/Acme.UI/Properties/PublishProfiles/LocalDev.pubxml.user [hunk 6]
```diff
@@ -13,10 +13,10 @@ by editing this MSBuild file. In order to learn more about this please visit htt
       <publishTime>08/08/2019 13:12:23</publishTime>
```
Comment (ettiennegous) [inline src/Acme.UI/Properties/PublishProfiles/LocalDev.pubxml.user]: Get rid of this file from Source Control

File: src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs (hunk 5 above)
Comment (KamranIE) [reply src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs]: Confirmed it will return all the list members.

**Pull request merged**